portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -17,31 +17,59 @@
 import com.google.gson.Gson;
 import java.util.Date;
 import java.io.IOException;
+import java.util.Arrays;
 import java.util.ArrayList;
 import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+// /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+// @WebServlet("/data")
+// public class DataServlet extends HttpServlet {
 
+//     
+// }
+
+
+/** Servlet that processes text. */
+@WebServlet("/text")
+public final class DataServlet extends HttpServlet {
+    //modify DataServlet to contain an ArrayList<String> variable
+    
+    private ArrayList<String> allComments = new ArrayList<String>();
+    
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        
-        //modify DataServlet to contain an ArrayList<String> variable
-        ArrayList<String> messages = new ArrayList<String>();
-        messages.add("This is the first message");
-        messages.add("This is the second message");
-        messages.add("This is the third message");
-        
+        response.setContentType("text/html");
```

Comment:
I'm a bit out of practice, but because you're sending json, i think `text/html` should instead be `application/json`. My guess is that change would be paired w/ a change to `getTextUsingArrowFunctions` to use `response.json` instead of `response.text`.

Reply:
Thank you! I'm fixing that